app/client/src/features/licences/AssociatedLicences.jsx

```diff
@@ -213,13 +213,33 @@ export default function AssociatedLicences({ licence }) {
     );
   }
 
-  function formatResultRow(result) {
+  function formatResultRow(result, licenceType) {
     const url = `${LICENSES_PATHNAME}/${result.id}`;
     return (
       <tr key={result.id}>
         <td className="text-nowrap">
           <Link to={url}>{result.licenceNumber}</Link>
         </td>
+        {licenceType === "VETERINARY DRUG" ? (
```

Comment:
May be able to juse use `{licenseType==="VETERINARY DRUG" & (...)}` without having to return null.  I'll approve anyway though.